flatters/flatten_cloud.go

```diff
@@ -0,0 +1,332 @@
+package flatters
```

Comment:
If the other package is called `expand`, shouldn't this package be called `flatten`?

Reply:
renamed